docs/getting-started.md

```diff
@@ -10,9 +10,9 @@ nav_order: 4
 
 | **Term** | **Description** |  
 | - | - |
-| chrultrabook | A modified Chromebook/box designed to run Windows, Linux, or even macOS |   
+| chrultrabook | A modified ChromeOS device designed to run Windows, Linux, or even macOS |   
 | coreboot     | UEFI firmware firmware replacement |  
-| UEFI         | The Unified Extensible Firmware Interface (UEFI) is a specification that defines a software interface between an operating system and platform firmware. UEFI replaces the legacy Basic Input/Output System (BIOS) firmware interface originally present in all IBM PC-compatible personal computers, with most UEFI firmware implementations providing support for legacy BIOS services. UEFI can support remote diagnostics and repair of computers, even with no operating system installed. (source: Wikipedia) |  
+| UEFI         | The Unified Extensible Firmware Interface (UEFI) is a new method by which OSes and platform firmware communicate, providing a BIOS alternative that uses only the information needed to launch the OS boot process. |  
```

Comment:
I feel like the UEFI description can still be simplified.

Reply:
im not sure how to, but i can delete like one word